python-squid/src/main/java/org/sonar/python/metrics/FileMetrics.java

```diff
@@ -21,7 +21,9 @@
 
 import com.sonar.sslr.api.AstNode;
 import java.util.ArrayList;
+import java.util.HashSet;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.HashSet'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-179&issues=AWD6HF7vEFfjxkQcPpio&open=AWD6HF7vEFfjxkQcPpio)

[ISSUE_KEY]: AWD6HF7vEFfjxkQcPpio